### ISSUE TITLE
Unablke to Download Models In Windows PC

### ISSUE THREAD

**Issue by AnatuGreen** [ohmplatform/FreedomGPT#84]
I recently came across FreedomGPT and I downloaded the app and installed it. I opened the app and I see the screen which I suppose is for downloading the offline AI model data of choice. The problem now is that I cannot even download any.  The download buttons are dead. The download button is not working

![image](https://github.com/ohmplatform/FreedomGPT/assets/95270336/97cf5d85-2526-4bc7-9294-f6e3f7fdd6a1)

Using Windows 11
512GB SSD memory, 16GB RAM.
AMD Ryzen 5, 4000 series

**Comment by johnnybpena1989** [comment]
yeah been struggling with this one since I first received the email announcing the release. Still no solution.

**Comment by shamator** [comment]
Yo consigo descargar los ficheros .bim, pero al dar seleccionar a cualquiera de ellos se queda eternamente diciendo Loading models y de ahí no pasa.

**Comment by VisseNekku** [comment]
Same here. Still no answer.

**Comment by TERIHAX** [comment]
same thing

**Comment by shamator** [comment]
Yo ya consigo que uncione on line, mas no responde, o el tiempo de demora es infinito.

**Comment by etherjack** [comment]
> It's because there's 0 GB available space. Apparently it can only install itself into C drive, which was obviously full because it's always full. I had to make room for it there for installation and then move it to a better drive.

Not sure if that's a joke or not, but I have > 55GB of free space on C:\ and am getting the same installation issues as all the others.

**Comment by insajd** [comment]
I had the same issue on Windows 11. I had it fixed, by right clicking file -> Properties -> Compatibility -> Run this program in compatibility mode for: Windows 8.
After that, I ran the .exe and it correctly identified 122 GB of free space.
